package.json

```diff
@@ -27,6 +27,7 @@
     "popper.js": "^1.16.1",
     "spotlight.js": "^0.7.8",
     "tippy.js": "^6.3.7",
-    "viem": "^1.6.0"
+    "viem": "^1.6.0",
+    "wagmi": "^1.3.10"
```

Comment:
Is there a reason to include both `@wagmi/core` and `wagmi`?
And they also seem to have different versions (1.3.9 vs. 1.3.10)

Reply:
It seems that `wagmi` already includes `@wagmi/core` as dependency.